FirebaseAuth/Sources/Swift/Auth/AuthSettings.swift

```diff
@@ -21,17 +21,25 @@ import Foundation
   /** @property appVerificationDisabledForTesting
       @brief Flag to determine whether app verification should be disabled for testing or not.
    */
-  @objc public var isAppVerificationDisabledForTesting: Bool
+  @objc public var appVerificationDisabledForTesting: Bool
+  @objc public var isAppVerificationDisabledForTesting: Bool {
```

Comment:
What is the purpose of adding the backing variable?

Reply:
It's necessary to be compatible with the Objective C implementation which provided both.

Reply:
Ah, got it. Might be worth removing one of them in the next breaking change. I think I'd prefer `isAppVerificationDisabledForTesting`.

Reply:
One might be better for the setter versus the getter. This is just one of many API improvements we may want to consider going forward.

For now, I'd like to stick to the goal of making the transition from the ObjectiveC implementation to the Swift implementation as seamless as possible.